libs/application-generic/src/utils/buildBridgeEndpointUrl.ts

```diff
@@ -0,0 +1,19 @@
+import { createHash } from 'crypto';
+
+/*
+ * Creates a bridge endpoint url to be used for request from novu cloud to the local
+ * workflow definition
+ */
+export const buildBridgeEndpointUrl = (
+  apiKey: string,
+  baseAddress: string
+): string => {
+  return `${buildBridgeSubdomain(apiKey)}.${baseAddress}`;
+};
+
+/*
+ * Creates a bridge subdomain based on the apiKey provided
+ */
+export const buildBridgeSubdomain = (apiKey: string): string => {
+  return createHash('md5').update(apiKey).digest('hex');
+};
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** This hashing function is simple enough for us to duplicate across both places we need it, adding a dependency on `@novu/application-generic` will significantly bloat the bundle size of `create-novu-app`. Let's duplicate for now until we have a better home for these types of functions, accepting double maintenance cost.‏

Reply:
Right, agree, should I add a note somewhere to say that there should be parity between the 2 places?